utils/music_crawlers.py

```
@@ -376,26 +377,40 @@ async def _check_vip_status(self):
+            data = resp.json()
+            code = data.get('code')
+
+            if code != 200:
+                if code in (301, 302):
+                    logger.warning(f"[{self.platform_name}] 凭证失效 (code: {code}, 重定向)")
+                    self._cookie_invalid = True
+                    self._is_vip = False
+                    self._vip_checked = True
                 else:
-                    logger.info(f"[{self.platform_name}] 用户为普通账号，已登录但无 VIP")
+                    logger.warning(f"[{self.platform_name}] 接口异常 (code: {code})，允许重试")
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)